build_scripts/source_available_copyright.txt

```diff
@@ -0,0 +1,11 @@
+
+# Copyright (C) 2019-2021 Cognizant Digital Business, Evolutionary AI.
```

Comment:
I bet you didn't really start this in 2019.  Maybe 2020.

Reply:
True! I'll update and merge